migrations/Dockerfile.dev

```diff
@@ -1,4 +1,70 @@
 FROM postgres:9.6-alpine
+MAINTAINER Régis Belson <me@regisbelson.fr>
```

Comment:
Can we update the Maintainer to some email from MDS team?

Reply:
Thanks, as per our discussion, removed all Maintainer tags. 

FYI, Maintainer tags are deprecated now, replaced by Author tags.